docs/dev/table/index.md

```diff
@@ -44,33 +44,48 @@ The following dependencies are relevant for most projects:
 * `flink-table-api-java-bridge`: The Table & SQL API with DataStream/DataSet API support using the Java programming language.
 * `flink-table-api-scala-bridge`: The Table & SQL API with DataStream/DataSet API support using the Scala programming language.
 * `flink-table-planner`: The table program planner and runtime.
```

Comment:
Update that this is the old planner of before Flink 1.9 and still the recommended one?